src/tsd/HttpQuery.java

```diff
@@ -422,7 +422,9 @@ public void badRequest(final BadRequestException exception) {
       buf.append("\"}");
       sendReply(HttpResponseStatus.BAD_REQUEST, buf);
     } else if (hasQueryStringParam("png")) {
-      sendAsPNG(HttpResponseStatus.BAD_REQUEST, exception.getMessage(), 3600);
+      final StringBuilder buf = new StringBuilder(10 +
```

Comment:
There are more of these and we can just let them fall through to the default HTML response when they ask for a PNG.